pytorch_lightning/loops/base.py

```diff
@@ -33,6 +33,12 @@ class Loop(ABC):
 
     .. codeblock:: python
 
+        if is_restarting:
+            restore()
+            is_restarting = False
+        else:
+            reset()
+
```

Comment:
How many details do we want to include in this docs overview? 
Need to be careful what we want to show, we don't want to start duplicating the implementation into the docs.

The raw loop structure below I think is already good as it is to just understand how the flow goes and where the core component is.

Reply:
Removed !